client/web/src/notebooks/listPage/NotebookNode.tsx

```diff
@@ -61,11 +61,18 @@ export const NotebookNode: React.FunctionComponent<React.PropsWithChildren<Noteb
                 <span className="d-flex align-items-center mr-3">
                     {node.viewerHasStarred ? (
                         <Icon
+                            role="img"
+                            aria-label="Unstar Notebook"
```

Comment:
```suggestion
                            aria-label="You have starred this notebook"
```

These aren't buttons, but we can label these so that screen readers can still imply their states